addons/mail/tests/test_link_preview.py

```diff
@@ -21,9 +22,40 @@ def _patched_get_html(*args, **kwargs):
     </head>
     </html>
     """
+    # To handle chunks read on stream requests
```

Comment:
can we add a `<title/>` tag with a different title to check that `og:title` has the priority ?